internal/faustjs.org/docs/migrationPath/overview.mdx

```diff
@@ -2,6 +2,268 @@
+
+### Writing GraphQL Queries
+The biggest hurdle when migrating from GQty to Apollo is that now you will need to write queries manually.
+However this is not as bad as you think. You only need to identify the data that your pages and components need and try to request the appropriate fields.
+
+For instance take a look at the `PostInfo` component we posted before. It supposed to display the `Post` Information using the `author` and `date` fields.
+You can request those fields once the post or page fragment:
+
+```
+fragment PostsItemFragment on Post {
+    id
+    date
+    uri
+    title
+    author {
+    	node {
+        	name
+    	}
+    }
+}
+```
+
+This fragment is reusable and can be applied to any page query when requsting the Post data fields:
+
+```
+query GetPostsPage(
+    $first: Int!
+    $after: String
+  ) {
+    posts(first: $first, after: $after) {
+      edges {
+        node {
+          ...PostsItemFragment
+        }
+      }
+	}
+}
+```
+
+By following this approach, writing queries for your pages become easier as you will be able to find and change them easily when needed.
+
+### Using the WP Template system
+
+If you chose to use the new Faust WP Template Hierarchy, you will need to create individual components for each Template.
+Those components will be rendered according to the hierarchy rules, when you visit a page on the Headless website. More specifically you define:
+
+**front-page.js**: For rendering your site’s front page.
+**single.js**: For rendering a single post.
+**page.js**: For rendering a static page (page post-type).
+**category.js**: Representing category archive index pages.
+
+The full list of templates are available at [WordPress Docs](https://developer.wordpress.org/themes/basics/template-hierarchy/#the-template-hierarchy-in-detail).
+Bear in mind that not all types of templates are supported within Faust. You can check the dev console when you visit a page that gives you a hint of the available templates that this route will match:
+
+<img
+  src="/docs/img/possible-templates.png"
+  alt="The list of possible templates as shown in the dev console"
+/>
+
+You can see an example of the hierarchy in the [blueprint project](https://github.com/wpengine/atlas-blueprint-portfolio/tree/main/wp-templates) where we provided implementations for the main templates.
+
+## Overview of the changes needed in to migrate from GQty to Apollo
+
+Given the above information, we can now list an brief overview of the changes needed in to migrate from GQty to Apollo.
+You can also review the code of this [Blueprint Project](https://github.com/wpengine/atlas-blueprint-portfolio/tree/main) that shows how the end result will look like.
+
+**1. Capture and migrate Presentational Components**
+
+Create a list of components that can be reused into the new version of Faust and move them into the new projects folder.
+
+**2. Import Faust Dependencies**
+
```

Comment:
Do we want folks to remove any old `node_modules` first before the import?

Reply:
I think this one is implied.